interface/wx/clntdata.h

```diff
@@ -59,6 +59,50 @@ class wxClientDataContainer
 
 
 
+/**
+    @class wxSharedClientDataContainer
+
+    This class is a replacement for @ref wxClientDataContainer, and unlike
+    wxClientDataContainer the wxSharedClientDataContainer client data is
+    possible to copy (as a shared ptr) when instances of it are cloned.
+    Like wxClientDataContainer, wxSharedClientDataContainer is a mixin
+    that provides storage and management of "client data.". The client data
+    is reference counted and managed by the container.
+
```

Comment:
I think we also need to explain that changing the data used by any object changes it for all the other ones too here.

Reply:
I hope the recently added text is ok.